test/api/auctionrun.js

```diff
@@ -63,7 +63,7 @@ describe('Pubfood auction run api', function() {
 
     pf.setAuctionProvider({
       name: 'auctionProvider',
-      libUri: '../test/fixture/lib.js',
+      libUri: 'fixture/lib.js',
```

Comment:
The rampant "chore-like" changes to test files ought to be part of a separate pull request.

We can be somewhat lenient here due to the pervasive nature of changes to the event handling and improving the tests overall for use in nodejs and via browser testing with https://github.com/pubfood/pubfood/blob/master/gulpfile.js#L16 and https://github.com/pubfood/pubfood/blob/master/test/unit-test-index.html#L18